src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1403,7 +1392,7 @@ namespace WaterToAirHeatPumpSimple {
                     "Design Size Rated Air Flow Rate [m3/s]",
                     RatedAirVolFlowRateDes);
             } else {
-                if (simpleWatertoAirHP.RatedAirVolFlowRate > 0.0 && RatedAirVolFlowRateDes > 0.0 && !HardSizeNoDesRun) {
+                if (simpleWatertoAirHP.RatedAirVolFlowRate > 0.0 && RatedAirVolFlowRateDes > 0.0) {
```

Comment:
I see, this is already inside the `!HardSizeNoDesRun` if condition, so no need to extra check it.